wolfcrypt/src/sha256.c

```diff
@@ -1679,6 +1699,54 @@ void wc_Sha256Free(wc_Sha256* sha256)
 }
 
 #endif /* !defined(WOLFSSL_HAVE_PSA) || defined(WOLFSSL_PSA_NO_HASH) */
+#ifdef WOLFSSL_HASH_KEEP
+int _wc_Sha_Grow(byte** msg, word32* used, word32* len, const byte* in,
```

Comment:
Instead of duplicating this in both sha256.c and sha512.c, should this go into somewhere like hash.c?